docs/requirements/requirements.md

```diff
@@ -0,0 +1,310 @@
+# Requirements
+
+:::caution
+
+This document is in the making since we are still in the requirements engineering phase.
+
+:::
+
+## RE Wording in this Document
+
+- We use *Microservice* and *Service* interchangeably.
+- We use *Reference Architecture* and *System* interchangeably.
+- RA = reference architecture
+
+## Stakeholders
+
+- Prof. Dr. Steffen Becker -- The Examiner
+- Sandro Speth -- Supervisor
+- Niklas Meißner -- Supervisor
+- The Developers
+- S -- Researcher
+- F -- Researcher
+- SH -- Researcher
+
+## Requirement Categories
+
+- Constraints
+- Process
+  - EnPro -- The project itself
+  - Software Artefacts
+- Product
+  - Domain
+  - Usability
+  - Non-functional Requirements & Quality Attributes
+  - Performance
+- Design
+  - Architecture
+  - Microservice Patterns
+  - Loose Coupling
+  - Technologies
+  - Deployment
+
+## Constraints
+
+### R-C-001: No external services
+
+- We are not allowed to use external services (e.g., external microservices).
+- We are not allowed to add functional dependencies to other services.
+- Origin: Supervisors
+
+### R-C-002: The use of the system must be free of charge
+
+- Deployment included.
+- Origin: Sarah
```

Comment:
privacy concern: we do not have consent from all interviewed people
if we put their names publicly on the website, I guess we should at least ask them
imho anonymisation would be an option (just have aliases stakeholder 1, 2, ...)

Reply:
I agree. I simplified the stakeholders section which now only mentions the different roles (e.g., examiner, supervisor), see the latest commit.